at-paninij-core/at-paninij-proc/src/main/java/org/paninij/proc/factory/AbstractMessageFactory.java

```diff
@@ -76,7 +76,6 @@ public String buildQualifiedClassName() {
         packs.add("javax.annotation.Generated");
         packs.add("org.paninij.runtime.Panini$Future");
         packs.add("org.paninij.runtime.Panini$Message");
-        packs.add(ret.packed());
```

Comment:
This looks fishy to me. Why was this removed? Isn't this needed for arrays? Is the discovery of types to be imported using the type collector already work for arrays and generics?